plugins/grpc/java/v1.67.1/buf.plugin.yaml

```
@@ -0,0 +1,25 @@
+version: v1
+name: buf.build/grpc/java
+plugin_version: v1.67.1
+source_url: https://github.com/grpc/grpc-java
+integration_guide_url: https://grpc.io/docs/languages/java/quickstart
+description: Generates Java client and server stubs for the gRPC framework.
+deps:
+  - plugin: buf.build/protocolbuffers/java:v25.3
```

```suggestion
  - plugin: buf.build/protocolbuffers/java:v28.2
```

This should now be compatible since 27.4, but I'll do some manual testing with generated SDKs before merging.